README.md

```diff
@@ -331,6 +333,58 @@ Drop data for Cetus bounty missions. [(Source)](https://n8k6e2y6.ssl.hwcdn.net/r
 ]
 ```
 
+### /data/zarimanRewards.json
+
+Drop data for Zariman bounty missions. [(Source)](https://n8k6e2y6.ssl.hwcdn.net/repos/hnfvc0o3jnfvc873njb03enrf56.html#zarimanRewards)
+
+#### Data structure:
+```json
+"zarimanRewards": [
+  {
+    "_id": "255771097c27769a3a76bfa38c9fba49",
+    "bountyLevel": "Level 50 - 55 Zariman Bounty",
+      "rewards": {
+        { ... }
+        "C": [
+          {
+            "_id": "d5ccc328b76d91df416cc361d2209c1f",
+            "itemName": "15,000 Credits",
+            "rarity": "Uncommon",
+            "chance": 17.39,
+            "stage": "Final stage"
+          },
+        { ... }
+      ],
+    }
+  },
+  { ... }
+]
+```
+
+### /data/syndicates.json
+
+Drop data for syndicate vendors. Generated by data from the [Warframe Wikia](https://warframe.fandom.com/wiki/Syndicate).
+
+#### Data structure:
+```json
+{
+  "syndicates": {
+    "Steel Meridian": [
+      {
+        "_id": "d765637f1f1ae09add1b5265f5d29ea1",
+        "item": "Steel Meridian Sigil",
+        "chance": 100,
+        "rarity": "Common",
+        "place": "Steel Meridian (Brave) (1000 standing)",
+        "standing": "1000"
```

Comment:
standing can also be a number, since that'd make a bit more sense for a numerical value